davenmathews/Bright_Events
# #152972342 Implement Event Search

## This pull request implements event search

This is on top of other previous commits

**The tasks completed are**
GET /api/v1/events/?search=keyword

**This can be manually tested by:**
1. Clone the repository.
2. Use virtualenv to start a virtual environment.
3. Ensure the users.json and events.json are placed in your virtualenv folder and that they are empty
3. Install the requirements.txt.
4. Run appv1.py and open the local host in your browser.
5. Register new users and log in.
6. Create multiple events
7. Search for them with many combinations, the events should show up according to the keywords.

**Constrains:**
None: Both anonymous users and registered users can search for events

**[Finishes|Delivers #152972342]**
